eng/scripts/README.md

```diff
@@ -0,0 +1,157 @@
+| `--help` | `-h` | Show help message | |
+
+### PowerShell Script (`get-aspire-cli.ps1`)
+
+| Parameter | Description | Default |
+|-----------|-------------|---------|
+| `-OutputPath` | Directory to unpack the CLI | `./aspire-cli` |
+| `-Channel` | Channel of the Aspire CLI to download | `9.0` |
+| `-BuildQuality` | Build quality to download | `daily` |
+| `-OS` | Operating system (auto-detected if not specified) | auto-detect |
+| `-Architecture` | Architecture (auto-detected if not specified) | auto-detect |
+| `-KeepArchive` | Keep downloaded archive files after installation | `false` |
+| `-Help` | Show help message | |
+
+## Output Path Parameter
+
+The `--output-path` (bash) or `-OutputPath` (PowerShell) parameter specifies where the Aspire CLI will be unpacked:
+
+- **Default behavior**: Creates an `aspire-cli` directory in the current working directory
+- **Custom path**: You can specify any directory path where you want the CLI installed
+- **Directory creation**: The scripts will automatically create the directory if it doesn't exist
+- **Final location**: The CLI executable will be placed directly in the specified directory as:
+  - `aspire` (on Linux/macOS)
+  - `aspire.exe` (on Windows)
+
+### Example Output Paths
+
+```bash
+# Default - creates ./aspire-cli/aspire
+./get-aspire-cli.sh
+
+# Custom path - creates /usr/local/bin/aspire
+./get-aspire-cli.sh --output-path "/usr/local/bin"
+
+# Relative path - creates ../tools/aspire-cli/aspire
+./get-aspire-cli.sh --output-path "../tools/aspire-cli"
+```
+
+## Usage Examples
+
+### Bash Script Examples
+
+```bash
+# Basic usage - download to default location (./aspire-cli)
+./get-aspire-cli.sh
+
+# Specify custom output directory
+./get-aspire-cli.sh --output-path "/usr/local/bin"
+
+# Download with verbose output
+./get-aspire-cli.sh --verbose
+
+# Keep the downloaded archive files for inspection
+./get-aspire-cli.sh --keep-archive
+
+# Force specific OS and architecture (useful for cross-compilation scenarios)
+./get-aspire-cli.sh --os "linux" --architecture "x64"
+
+# Combine multiple options
+./get-aspire-cli.sh --output-path "/tmp/aspire" --verbose --keep-archive
+```
+
+### PowerShell Script Examples
+
+```powershell
+# Basic usage - download to default location (./aspire-cli)
+.\get-aspire-cli.ps1
+
+# Specify custom output directory
+.\get-aspire-cli.ps1 -OutputPath "C:\Tools\Aspire"
+
+# Download with verbose output
+.\get-aspire-cli.ps1 -Verbose
+
+# Keep the downloaded archive files for inspection
+.\get-aspire-cli.ps1 -KeepArchive
+
+# Force specific OS and architecture
+.\get-aspire-cli.ps1 -OS "win" -Architecture "x64"
+
+# Combine multiple options
+.\get-aspire-cli.ps1 -OutputPath "C:\temp\aspire" -Verbose -KeepArchive
+```
+
+## Supported Runtime Identifiers
+
+The following runtime identifier (RID) combinations are available:
+
+| Runtime Identifier | AOT Support |
+|-------------------|-------------|
+| `win-x64` | ✅ |
+| `win-arm64` | ✅ |
+| `win-x86` | ❌ |
```

Comment:
Why aren't these 3 available?

Reply:
- `win-x86` - Native AOT doesn't support this
- the others I need to explore using the corresponding arch-specific agents so we can AOT, so that's for a future PR.

I can add comments here for that.

Reply:
It doesn't need to be AOT'd for us to have a self-contained exe. We still build for win-x86, we just don't native AOT it.